Cargo.toml

```diff
@@ -37,13 +37,14 @@ debug = true
 name = "vector"
 section = "admin"
 maintainer-scripts = "distribution/debian/scripts/"
-conf-files = ["/etc/vector/vector.toml", "/etc/default/vector"]
+conf-files = ["/etc/vector/vector.toml"]
 assets = [
   ["target/release/vector", "/usr/bin/", "755"],
   ["config/vector.toml", "/etc/vector/vector.toml", "644"],
   ["config/examples/*", "/etc/vector/examples/", "644"],
+  ["config/agent/*", "/etc/vector/agent/", "644"],
+  ["config/aggregator/*", "/etc/vector/aggregator/", "644"],
   ["distribution/systemd/vector.service", "/lib/systemd/system/vector.service", "644"],
-  ["distribution/systemd/vector.default", "/etc/default/vector", "600"]
```

Comment:
SystemD is rather anti-env files, so I decided to remove our default usage of it. It can still be used via override conf files.